brother_printer_fwupd/utils.py

```diff
@@ -55,7 +64,7 @@ def __exit__(self, exc_class, exc, tb):
         if not exc_class or exc_class in (SystemExit, KeyboardInterrupt):
             return
 
-        if isinstance(exc, ExceptionGroup):
+        if sys.version_info >= (3, 11) and isinstance(exc, ExceptionGroup):
```

Comment:
Does this work as expected for Python < 3.11? I'd expect a `NameError: name 'ExceptionGroup' is not defined`

Reply:
It does, because `and` is short-circuiting and the right side is evaluated if and only if the left side `== True`.

Would you prefer something like `if hasattr(__builtins__, 'ExceptionGroup'):` which avoids hard-coding a specific version?

Reply:
Finessed a bit in 974c39e9e22942db504b6030d496119b8ded7498